awssql/driver_infrastructure/pg_database_dialects.go

```diff
@@ -349,32 +349,31 @@ func (m *AuroraPgDatabaseDialect) GetLimitlessRouterEndpointQuery() string {
 	return "select router_endpoint, load from aurora_limitless_router_endpoints()"
 }
 
-type RdsMultiAzDbClusterPgDialect struct {
+type RdsMultiAZClusterPgDatabaseDialect struct {
```

Comment:
nit: can we make AZ -> Az (So RdsMultiAzClusterPgDatabaseDialect) to be consistent? just cause all the acronyms in this are all lower-cased after the first letter like RDS -> Rds PG -> Pg?